server/processors/PlayerMessageProcessor.cpp

```diff
@@ -42,7 +42,7 @@ void PlayerMessageProcessor::playerMessage(PlayerColor player, const std::string
 	if (handleCheatCode(message, player, currObj))
 	{
 		if(!gameHandler->getPlayerSettings(player)->isControlledByAI())
-			broadcastSystemMessage(VLC->generaltexth->allTexts[260]);
+			broadcastSystemMessage("~~core.genrltxt.260~~");
```

Comment:
If you need client-side translations you should send MetaString to client instead

Reply:
And change the messaging logic?

Reply:
Why not?
Existing messages with player input can be added as raw strings to MetaString.
Better than these `~~` hacks